src/Parlot/Fluent/Parser.Compile.cs

```diff
@@ -5,22 +5,31 @@
 
 namespace Parlot.Fluent
 {
-    public abstract partial class Parser<T>
+    public partial class Parsers
```

Comment:
Like there is `public static class ParserExtensions` for `TryParse` it should be a `ParserExtensions` for `Compile`. Maybe both as partial classes.